internal/pkg/config/config.go

```diff
@@ -70,6 +71,8 @@ var ConfigKeys = []string{
 	SKECustomEndpointKey,
 }
 
+var FolderPath string
```

Comment:
I think this `var` could be private:

```suggestion
var folderPath string
```